client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/GroupedInvestigations/GroupedInvestigationForm/NoContactsMessage/NoContactsMessage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Typography } from '@material-ui/core'
+import React from 'react'
+    
+const message = 'אין מגעים לחקירות המשותפות';
+
+const NoContactsMessage = () => {
+    return (
+        <Typography variant='h5'>
+            {message}
+        </Typography>
+    )
+}
+
+export default NoContactsMessage
```

Comment:
; at the end

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1233/commits/c0ae3e23e053f718fe8807595a9df1a065eb9c9b)